git-hub

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+import subprocess 
+from github import Github 
+import yaml
+import click
+import getpass
+
+
+@click.group()
+def cli():
+	pass
+
+@cli.command()
+@click.argument("command", default = "")
+@click.argument("num", default = 0)
+def hub(command,num):
+	if command == "pr":
+		#runs git and gets user and repo of current folder.
```

Comment:
According to the Click documentation(http://click.pocoo.org/5/arguments/) under "Variadic Arguments", I can only use one parameter that has unlimited arguments(by using nargs = -1). I also tried using *args and **kwargs and it doesn't accept keywords since I need to put the string into @click.arguments() to specify the parameters before the function hub. I can get it to say:
```
@click.argument("command", default = "")
@click.argument("args", nargs= -1)
def hub(command, args):
	if command == "pr":
		pr_num = int(args[0])
		pr(pr_num)		
	else:
                ...
```
I'm not sure how else to approach it since I have to specify the parameters beforehand in @click.argument(args) and it doesn't take keywords in the parameters.